specutils/analysis/correlation.py

```diff
@@ -49,6 +50,11 @@ def template_correlate(observed_spectrum, template_spectrum, lag_units=_KMS,
         ``template_logwl_resample(spectrum, template, delta_log_wavelength=.1)``.
         If False, *no* resampling is performed (and the user is responsible for
         a sensible resampling).
+    mode: str
+        If you choose FFT method, the correlation will be done through the use
+        of convolution and will be calculated faster (for small spectral 
+        resolutions it is often correct). In other cases, the direct method
+        from the numpy will be used.
```

Comment:
```suggestion
    method: str
        If you choose "FFT", the correlation will be done through the use
        of convolution and will be calculated faster (for small spectral 
        resolutions it is often correct), otherwise the correlation is determined
        directly from sums (the "direct" method in `~scipy.signal.correlate`).
```